sumologic/resource_sumologic_cse_custom_entity_type_test.go

```diff
@@ -0,0 +1,110 @@
+package sumologic
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccSumologicSCECustomEntityType_create(t *testing.T) {
+	var customEntityType CSECustomEntityType
+	nName := "New Custom Entity Type"
+	nIdentifier := "identifier"
+	nFields := []string{"field1"}
+	resourceName := "sumologic_cse_custom_entity_type.custom_entity_type"
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCSECustomEntityTypeDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testCreateCSECustomEntityTypeConfig(nName, nIdentifier, nFields),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckCSECustomEntityTypeExists(resourceName, &customEntityType),
+					testCheckCustomEntityTypeValues(&customEntityType, nName, nIdentifier, nFields),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+				),
+			},
+		},
+	})
+}
```

Comment:
add tests for import and update?

Reply:
added update test under
https://github.com/SumoLogic/terraform-provider-sumologic/pull/275/commits/eaf35c35121c9da0e0db4f53f0ffe2d8c937c064